src/app/community/create/create.component.html

```diff
@@ -0,0 +1,27 @@
+<mat-card class="communityCreationForm">
+  <mat-card-title>Community Creation Form</mat-card-title>
+
+  <mat-card-content>
+    <mat-form-field id="communityNameField">
+      <input matInput placeholder="Community Name" [formControl]="nameFormControl" [(ngModel)]="nameText" maxlength="25" >
```

Comment:
I think the placeholders could just be Name (shorthand form, used in URL), Title, Description.

Reply:
ok sure. Also, why can't we use Name as the sidebar nav text? It makes sense to me that they're the same since name is used in the url... o.o

Reply:
The sidebar nav text is basically just an area for the creator to put anything they want relevant to the community such as rules, etc...

Reply:
oh, I thought that was description. Uh... I'm not convinced that our user know will the difference... Like if in the form they have "Description" and "Sidebar Navigation Text", I think they'll just fill out description and not know what side bar nav text means...